### PR TITLE
Contract display

### PR DESCRIPTION
**Pull request by andrewcgraves**
https://user-images.githubusercontent.com/25449011/195521270-580c9334-ef23-4bce-bfa7-992dcf8404c2.mp4

Created a way to view contracts and associated users. Right now it displays all users but should be further modified to associate a user with a contract with a checkmark or something

### CODE REVIEW

**Comment by andrewcgraves** [comment]
im going to bed im tired

File: Assets/Scripts/GameManager/GameController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        contractController.CreateNewContract("Contract4", 100, 10, 1);
+        contractController.CreateNewContract("Contract5", 100, 10, 1);
+        contractController.CreateNewContract("Contract6", 100, 10, 1);
+        contractController.CreateNewContract("Contract7", 100, 10, 1);
+        contractController.CreateNewContract("Contract8", 100, 10, 1);
+        contractController.CreateNewContract("Contract9", 100, 10, 1);
+
+        employeeController.CreateNewEmployee("Andrew Graves", 10000, null);
+        employeeController.CreateNewEmployee("Andrew Graves2", 10000, null);
+        employeeController.CreateNewEmployee("Andrew Graves3", 10000, null);
```
Comment (jalexcrawford) [inline Assets/Scripts/GameManager/GameController.cs:25]: Creative naming :P

**Review by jalexcrawford — APPROVED** [review]
Looks good! Approved.

**Review by johnavalos2 — APPROVED** [review]

**Comment by Frances637** [comment]
Nice Contract layout! I feel that the Nokia font really works for our game aesthetic.

**Pull request merged**